CompetitionNunit/CompetitionNunit/Tests/CertificationTest.cs

```diff
@@ -0,0 +1,123 @@
+﻿using AventStack.ExtentReports;
+using CompetitionNunit.TestModel;
+using CompetitionNunit.Pages;
+
+
+using CompetitionNunit.Utilities;
+using NUnit.Framework;
+
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CompetitionNunit.Tests
+{
+    [TestFixture]
+    public class CertificationTest : CommonDriver
+    {
+
+        CertificationPage certificationPageObj;
+#pragma warning disable
+        public CertificationTest()
+        {
+
+            certificationPageObj = new CertificationPage();
+        }
+
+
+        [Test, Order(1)]
+        public void TestAddCertification()
+
+        {
+            certificationPageObj.CertificationTabClick();
+            Thread.Sleep(2000);
+            certificationPageObj.clearExistingdata();
+
+            //read test data from the json file
+            List<CertificationTestModel> AddCertificationfile = JsonHelper.ReadTestDataFromJson<CertificationTestModel>("F:\\CompetitionTask\\Task2Nunit\\CompetitionNunit\\CompetitionNunit\\JsonFile\\AddCertificationfile.json");
+            Console.WriteLine(AddCertificationfile.ToString());
+            foreach (var data in AddCertificationfile)
+            {
+                string Certificate = data.Certificate;
+                Console.WriteLine(Certificate);
+
+                string CertifiedFrom = data.CertifiedFrom;
+                Console.WriteLine(CertifiedFrom);
+
+                string Year = data.Year;
+                Console.WriteLine(Year);
+
+                certificationPageObj.AddCertification(Certificate, CertifiedFrom, Year);
+                string newCertificatefile = certificationPageObj.getnewCertificatefile();
+
+                if (Certificate == newCertificatefile)
+                {
+                    Assert.That(Certificate, Is.EqualTo(newCertificatefile));
+                }
+                else
+                {
+                    Console.WriteLine("Check error");
```

Comment:
so there is one thing i want you to think about! I saw that you are clearing the test data at the start but should do this for before any tests run and then delete only the test data the test adds after the run, if we can a look at the onboarding example

e.g before adding language you need to make sure there are no existing languages as this could result in false positive test pass
you will need to think about this for all the scenarios, this should be done before you start any tests 

the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing language so that the state is reset, this will guarantee that your test does not produce false negatives and passes always. You can not assume that test execution order is not always the same, imagine if you run the tests  in parallel mode the all tests will run at the same time so you need to also make sure you do not wipe out other tests data
hence why when you remove data that other tests are using.  The best thing to do is delete only data you added for the test to be correct
e.g test 1 added english and french as language test 2 added hindi then when you reset data you already know what language you are adding for each test
there for on reset of the two test you should delete english and french for test 1 and hindi for test 2

So to sum up you should delete all data before the start of any test and then after each test you should delete the data that a particular test is adding!
This is ensure that upon staring the test pack you have reset state and upon finishing each test deletes their data that they added!
Think about first clean up before any test runs (you delete all data around this feature e.g  before any language test run you delete all languages). Then after each test delete only data that the test has added! (test1 added english and test2 to added french then test1 should only delete english and test2 should delete french only) 

also say you are updating a language then you must added language then update it do not assume the language is there! This applies for delete or any scenario